themes/digital.gov/layouts/resources/single.html

```diff
@@ -1,73 +1,49 @@
-
-<main role="main" id="main-content">
-  <article>
-    <header>
-      <div class="grid-container grid-container-desktop">
-        <div class="grid-row">
-          <div class="grid-col-12" data-edit-this="{{- $path -}}">
-            <p class="breadcrumb">
-              <a href="{{- "resources/" | absURL -}}" aria-label="Back to Resources landing page">
-                <svg class="usa-icon dg-icon dg-icon--standard margin-bottom-05" aria-hidden="true" focusable="false">
-                  <use xlink:href="{{ "uswds/img/sprite.svg#arrow_back" | relURL }}"></use>
-                </svg>
-                <span>All Resources</span>
-              </a>
-            </p>
-            {{- if .Params.kicker -}}
-            <p class="kicker">{{- .Params.kicker -}}</p>
-            {{- end -}}
-            {{/* Page Title */}}
-            <h1>{{- .Title | markdownify -}} </h1>
-
-            {{/* Deck */}}
-            {{- if .Params.deck -}}
-            <div class="deck">{{- .Params.deck | markdownify -}}</div>
-            {{- end -}}
-          </div>
+  {{ $path := "" }}{{ with .File }}{{ $path = .Path }}{{ else }}{{ $path = .Path }}{{ end }}
+  <main role="main" id="main-content">
+    <article class="grid-container grid-container-desktop">
+      <header>
+        <div data-edit-this="{{- $path -}}">
+          <p class="breadcrumb">
+            <a href="{{- "resources/" | absURL -}}" aria-label="Back to Resources landing page">
+              <svg class="usa-icon dg-icon dg-icon--standard margin-bottom-05" aria-hidden="true" focusable="false">
+                <use xlink:href="{{ "uswds/img/sprite.svg#arrow_back" | relURL }}"></use>
+              </svg>
+              <span>All Resources</span>
+            </a>
+          </p>
+          {{- if .Params.kicker -}}
+          <p class="kicker">{{- .Params.kicker -}}</p>
+          {{- end -}}
+          {{/* Page Title */}}
+          <h1>{{- .Title | markdownify -}} </h1>
+
+          {{/* Deck */}}
+          {{- if .Params.deck -}}
+          <div class="deck">{{- .Params.deck | markdownify -}}</div>
+          {{- end -}}
         </div>
-      </div>
-    </header>
+      </header>
 
-    <section>
-      <div class="grid-container grid-container-desktop">
-        <div class="grid-row tablet-lg:grid-gap-6">
-
-          <div class="grid-col-12 tablet:grid-col-9" data-edit-this="{{- $path -}}">
-
-            {{/* Content */}}
-            <div class="content usa-prose">
-              {{- .Content -}}
-            </div>
-
-            {{- partial "core/page-data.html" (dict "page" . "page_type" "resource" "branch" ($.Scratch.Get "branch")) -}}
-
-          </div>
-
-          <div class="grid-col-12 tablet:grid-col-3">
-
-            {{/* Table of contents */}}
-            {{- partial "core/toc.html" . -}}
-
-            {{/* Last updated */}}
-            {{/* {{- partial "last-updated.html" . -}}  */}}
+      <div class="usa-in-page-nav-container">
```

Comment:
The unique requirements of this template is causing sidenav headers to appear as in-page-nav items.

![image](https://user-images.githubusercontent.com/3385219/219107808-5a0bcc23-a17d-4c45-96b6-3881491078a8.png)

Right now we're hiding them, but as we're cleaning up these templates we should fix this layout issue. Meaning, we separate `<main>` from the in-page nav `<aside>`.

Reply:
We should discuss this, I'm curious how we can structure the markup.